tests/usbus_board_reset/main.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2022 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @file
+ * @brief       
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ *
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "shell.h"
+
+#include "usb/usbus.h"
+#include "usb/usbus/cdc/acm.h"
+
+#if MODULE_STDIO_RTT
+#include "xtimer.h"
```

Comment:
Is it needed ?
If yes, ztimer should be preferred I guess.

Reply:
It's just a leftover. Fixed.

Reply:
xtimer is still there on my side. Perhaps you miss a commit ?

Reply:
Indeed.